cmd/krew/cmd/install.go

```diff
@@ -62,6 +99,20 @@ Remarks:
 			var pluginNames = make([]string, len(args))
 			copy(pluginNames, args)
 
+			// Downloads manifest file from given URL
+			if *manifestURL != "" {
+				fileName, err := downloadFile(*manifestURL)
+				if err != nil {
+					fmt.Fprintln(os.Stderr, "Error downloading file")
```

Comment:
Please do
```go
return errors.Wrapf(err, "Error downloading manifest from %q", *manifestURL)
```

Just logging an error won't be enough ;)

---

Please also put some thought into the question whether the cleanup function (remove the temp-file) should be called when an error is returned here. Otherwise we might leak temporary files.

Reply:
Done. I made a call to cleanup function to clear the temp files.